migrations/postgres/006_consensus.up.sql

```diff
@@ -0,0 +1,25 @@
+
+ALTER TABLE node_record ALTER COLUMN enr TYPE VARCHAR(1000);
```

Comment:
expanding because i think technically 304 is max length in enr specification but I've noticed before people append query strings on it lmao. no real need to cap it at the db layer imo